src/bus_monitor.cpp

```diff
@@ -10,6 +10,84 @@
 namespace panel
 {
 
+static void resetLEDState()
+{
+    std::cout << "Reset LED state" << std::endl;
+    std::vector<std::string> ledsPathOnBasePanel{
```

Comment:
> Santosh: Please check if this can be constexpr. We don't want to create the vector every time in the function

> Sunny: This function will be called only when present property of Everest base panel will be set from false to true i.e. only at panel CM and at the time of system coming up.
How frequently this will be called? Just trying to understand if it will have much effect?

Reply:
Just good coding practice?

Reply:
BTW, don't take the constexpr literally, you cannot create a local std::vector that is a constexpr. You can however, do something like a std::array<const char*>

Reply:
Yup.